### ISSUE TITLE
Undefined functoin "lfmComputeUpsilonMatrix"  

### ISSUE THREAD

**Issue by ymohit** [SheffieldML/multigp#1]
This particular function is missing from the current package. For this part i think it depends on older package named "Kern0p222" which contain one file named lfmComputeUpsilonMatrix.f (fortran file). This particular file is not working even after following given instructions on webpage of this package. Please suggest what is the best way to go ? 

I would prefer to write a matlab code for this if somebody can help me understanding the mathematical functionality of this particular function.

**Comment by lawrennd** [comment]
Hi,

I've added the old kernel toolbox to github, I'm going to restructure the GPmat toolbox soon to depend on updated versions of the old toolboxes (given time!). In the meantime you can find the old kern toolbox here:

http://github.com/lawrennd/kern/

**Comment by ymohit** [comment]
Thanks a lot !!! Although tI have rewritten fortran files (from Kern toolbox available on your website). Still i think it will help me to verify what i did and also increases usability of this package. One more suggestion that will help me and other users is to have a working demo of LFM on some toy data sets. As of now none of demo code really works simply because all data files are missing.

**Comment by lawrennd** [comment]
Yes, that would be good. I don't know if one will emerge as part of the move of the old repos to github ... one problem is that we are reluctant to rewrite code that isn't being released openly, perhaps once it's on github there will be more eyes on it!

**Comment by ymohit** [comment]
Based on my little understanding I don't think you people have to write much extra lines of codes as you already have "demLfmToy.m".  All you need is to put data file and make changes of path in the this script.  It will be nice if you upload data that you have used in your paper e.g. motion capture data in Latent Force Models (http://jmlr.csail.mit.edu/proceedings/papers/v5/alvarez09a/alvarez09a.pdf). Despite giving introduction it is not very much clear which dimension is being used as latent force which as observation variables.

**Comment by lawrennd** [comment]
I don't seem to have that file locally, @maalvarezl do you have these mat files?

`datasetODE31_150_5Ind0.mat`

`datasetODE31_100_5Ind1.mat`

These are two demos that were never in the original release (check the `%/~` comments either side). So it may be that we don't have the rights to release them. I don't recall what they contained.

**Comment by lawrennd** [comment]
Just bumping on this @maalvarezl do you have these mat files?
